### ISSUE TITLE
LFO Operation-level Emission Summary Page

### ISSUE THREAD

**Issue by suhafa** [bcgov/cas-reporting#202]
#### Description:

For LFO's who have multiple facilities, they will also want to see their emissions data aggregated up from all facility reports into a higher-level "Operation-level" emissions summary page. 

We already have API/service files for [emission category aggregation by facility_id](https://github.com/bcgov/cas-registration/blob/develop/bc_obps/reporting/service/emission_category_service.py#L39).
We'll need to add similar functiions that aggregate by emission category at the operation level.

It's possible we can re-use the json schema from the facility emission summary page.

This is likely to be used in verification so may need to be reviewed/tested by verifiers. 

#### Acceptance Criteria:

Given that I am a LFO Industrial Reporter
When I have completed all my facility-level data entry for the annual report
Then I want to see an aggregated (operation-level) emissions summary page so that I can see all the totals together. 

[Wireframe](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=17438-3471&t=1ZNIH6bcaqHB63jD-4)

#### Development Checklist:

- [ ] A frontend page for the operation level emission summary as per the wireframe
- [ ] An API route to fetch the operation-emission-summary aggregated data
- [ ] An API service that aggregates the emissions at the operation level
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Blocked By
#332 
#333 
#345 

#### Notes:

-

**Comment by dleard** [comment]
@patriciarussellCAS @pbastia @suhafa blocked until we receive the "rules" about how to apply emission categories based on the data we collect. Adding the attn: Business Area tag

**Comment by rdromey** [comment]
@dleard to refine dev checklist

**Comment by patriciarussellCAS** [comment]
From Sprint planning: add a feature to this form that warns the users when there are still incomplete facility-level emission summary pages. That way, they understand that the aggregate totals they see on this form are not final yet.

**Comment by fviduya** [comment]
I have tested this in the DEV environment, [here is the test document for reference](https://bcgov-my.sharepoint.com/:w:/g/personal/francesca_viduya_gov_bc_ca/EZnmU0bBA4xLsoJux3eTYtUB8MMxDfhgyk-LIuhKRcwwBA?e=NQ8Tgm).

Currently, we do not have a path to add emissions across multiple facilities while filling out an LFO,  so the data were entered in an SFO operation first and verified that expected data is displayed to show that it is working as expected.

I was able to test different Reporting Activities to yield values for different Emission categories, specifically:
- Fugitive Emissions
- Industrial Process Emissions
- On-site transportation Emissions
- Emissions from waste

As for the other categories, I was not able to test them as the activities are still in development. Documented as well are the reference ticket numbers of each activities that I was not able to use.

I can verify that the expected values for the available emission categories are displayed in the Emission Summary page based on the emission data I used in the activities. The data can be seen in the document attached. 
**Update:** The Operation Emission Summary page shows the same data as the Emission Summary page as well

Also wanted to note that I couldn't find a ticket for Petrochemical Production, not sure if I was just not able to see it myself or if the ticket doesn't exist yet.

Moving to Pending PO for review. cc: @patriciarussellCAS / @vesselak

**Comment by patriciarussellCAS** [comment]
Hey @acatchpole - is there a way to properly test this page in Dev without the other parts of the LFO workflow in place? I assume we can't but wanted to double check. 
@fviduya - what you tested was the SFO emissions summary page, which is not the same thing. There is a second summary page that is shown only to LFOs later in the workflow and it should be an aggregate of all the facility-level emission summaries. I don't think we can properly test this until we have the ability to add more than one facility emissions report. 

For the Petrochemical Production - we have not created a ticket for this because at this point there are NO operations in BC that do this activity, so it's not in scope for MVP.

**Comment by acatchpole** [comment]
You are correct: there is currently no way to manually verify that emissions across an operation are properly collected. You can use an SFO, add emissions, check the facility emission summary, and then manually enter the URL for the operation emission summary page and see that it shows the same stuff. This shows that the page 'works' (renders and shows some numbers). And then there are tests that show that the functions that get those numbers DO properly summarize facility emissions across an operation. 

But, again, yes, there is currently no way to see that for yourself.

**Comment by fviduya** [comment]
Hi @patriciarussellCAS, based on @acatchpole's comment, I've checked the Operation Emission Summary page, updated my comment above, and the test document I've attached here. I can verify that the Operation Emission Summary page shows the same data, and that it renders as well. Moving to Pending PO for review.

**Comment by patriciarussellCAS** [comment]
Thanks for double checking that @fviduya - I think I will accept this ticket for now, but will make a note that we'll need to re-test this once the full LFO workflow is complete (which we plan to do through UAT once it's all ready) :)
